app/src/main/java/com/example/gradeassist/RetrofitClientInstance.kt

```diff
@@ -1,28 +1,28 @@
 package com.example.gradeassist
 
-//import retrofit2.Retrofit
-//import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
 
     // RetorofitClientInstance
 
-//    private var retrofit: Retrofit? = null
-//    private val BASE_URL = "https://pkgstore.datahub.io"
-//
-//    val retrofitInstance: Retrofit?
-//        get() {
-//
-//            //if retro is not created
-//            if (retrofit == null) {
-//
-//                // create it
-//                retrofit = retrofit2.Retrofit.Builder()
-//                    .baseUrl(BASE_URL)
-//                    .addConverterFactory(GsonConverterFactory.create())
-//                    .build()
-//            }
-//
-//            return retrofit
-//        }
```

Comment:
not a good idea to leave commented code in commits

Reply:
I agree.  Instead of commenting out code, my suggestion is:
1) Use feature branches in Git to keep experimental work separate from master.
2) If you're removing something that used to be there, just remove it entirely.  You can always go back in history to see what used to be there.